modules/ROOT/pages/gov-cloud-security.adoc

```diff
@@ -0,0 +1,55 @@
+= Security in Government Cloud
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Government Cloud meets the security requirements required by FedRAMP (Federal Risk and Authorization Management Program) and other security protocols.
+
+[NOTE]
+Some xref:anypoint-security::index.adoc[Anypoint Security] features are not available in this release of Government Cloud.
+
+== FedRAMP Compliance
+
+Government Cloud meets all FedRAMP security and compliance standards. FedRAMP provides a standardized approach to security assessment, authorization, and continuous monitoring for cloud products and services used by the US government.
+
+A Third-Party Assessment Organization (3PAO) performs the security assessments following guidance in the National Institute of Standards and Technology (NIST) 800-37 publication. The security assessment validates management, operational, and technical security controls used to protect the confidentiality, integrity, and availability of the MuleSoft service and the data it stores, transmits, and processes.
+
+MuleSoft obtained the FedRAMP Agency Authority to Operate (ATO) at the *Moderate Impact Risk* level.
+
+== Security protocols
+
+In addition to FedRAMP compliance, Government Cloud adheres to these security protocols:
+
+* *Federal Information Processing Standards* (FIPS 140-2) — Complies with hardware and software encryption standards for all data at rest (stored data).
+* *Transport Layer Security* (TLS) 1.2 encryption — Implements encryption for all external and internal data transmission to block “man in the middle” attacks.
+* *NIST 800-53* — Enhances physical and logical security.
+* *CIS benchmarks* — Hardens instances to address Center for Internet Security checklists.
+
+== Secure Hosting Platform
+
+Government Cloud is hosted in Amazon Web Services (AWS) High Authorized GovCloud. AWS GovCloud is a FedRAMP High Authorized, ITAR-compliant environment with provisional DoD IL2, IL4, and IL5 authorization.
+
+== Cloud Security and Data
+
+Government Cloud provides a highly secure environment for government entities to perform sensitive data manipulations.
+
+MuleSoft transmits but does not inspect, permanently store, or otherwise interact directly with government data. Government agencies always maintain control over their data, configuration, and workers.
+
+== Support
+
+MuleSoft supports Government Cloud with dedicated support personnel in the United States. MuleSoft provides continuous monitoring services to protect confidentiality, integrity, and availability of government data managed in Anypoint Platform.
+
+The MuleSoft security team:
+
+* Follows industry best practices.
```

Comment:
"industry best practices" is pretty vague: for what? such as?

Reply:
Added such as